<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_gestaoVendas</class>
 <widget class="QDialog" name="fm_gestaoVendas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GestÃ£o de Vendas</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/imagens/img/images.png</normaloff>:/imagens/img/images.png</iconset>
  </property>
  <widget class="QTableWidget" name="tw_gv_listaVendas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>601</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_gv_listaprodutos">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>601</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_gv_relatorio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Gerar relatorio PDF</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Data inicial </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="de_gv_dataini"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>141</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Data final</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="de_gv_datafim"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_gv_filtar">
      <property name="text">
       <string>Filtrar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
